src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 200px;
+  font-style: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+html {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+.form-big {
+  position: relative;
+  min-width: 200vh;
+  margin-top: 20px;
+  background-position: center;
+}
+
+.img-big {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  height: 19px;
+  width: 19px;
+}
+
+.input-big::placeholder {
+  color: #3d4e61;
+}
+
+.input-big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  font-family: inherit;
+}
```

Comment:
Check this item of the checklist and try to rewrite your styles 
<img width="1095" alt="image" src="https://user-images.githubusercontent.com/99403198/216771207-3c07a29a-1dfe-49db-b9be-caec117fc895.png">